pandas/tests/test_panel.py

```diff
@@ -2,11 +2,14 @@
 # pylint: disable=W0612,E1101
 
 from datetime import datetime
-from inspect import getargspec
 import operator
 import nose
 from functools import wraps
-
+try:
+  from inspect import signature
+except:
```

Comment:
should be in ``compat/__init__.py``
``from inspect import getargspec as signature``for PY2
``from inspect import signature`` for PY3

then just ``from compat import signature`` and use it